validator/src/admin_rpc_service.rs

```diff
@@ -426,3 +448,36 @@ pub async fn connect(ledger_path: &Path) -> std::result::Result<gen_client::Clie
 pub fn runtime() -> jsonrpc_server_utils::tokio::runtime::Runtime {
     jsonrpc_server_utils::tokio::runtime::Runtime::new().expect("new tokio runtime")
 }
+
+pub fn load_staked_nodes_overrides(
+    path_or_url: &String,
+) -> std::result::Result<StakedNodesOverrides, Box<dyn error::Error>> {
+    debug!(
+        "Loading staked nodes overrides configuration from {}",
+        path_or_url
+    );
+    if Path::new(&path_or_url).exists() {
+        return try_config_from_local_file(path_or_url);
+    }
+    if let Ok(url) = Url::parse(path_or_url) {
+        return try_config_from_url(url);
+    }
```

Comment:
My preference is to remove the URL support entirely. Local files only. It's trivial for the caller to `wget` the file, or use whatever other standard url fetching tooling they prefer.

Reply:
+1, I changed for using only the path as you recommended.